docs/developer/app-store/apps/taxes/avatax/overview.mdx

```diff
@@ -123,11 +123,13 @@ The value of the `avataxTaxCalculationDate` field must be a valid UTC datetime s
 ### Customer code
 
 **Metadata key: `avataxCustomerCode`**
-**Object: `User`**
+**Object: `Checkout` or `Order`**
 
-If the user is logged in, the Taxes App will use the user's id as the [customer code](https://developer.avalara.com/erp-integration-guide/sales-tax-badge/transactions/simple-transactions/customer-codes/). If the user is not logged in, [a dummy customer code will be used](https://developer.avalara.com/ecommerce-integration-guide/sales-tax-badge/transactions/simple-transactions/).
+You can add `avataxCustomerCode` to your user `Checkout` or `Order` metadata. Avatax app will then send it as the [customer code](https://developer.avalara.com/erp-integration-guide/sales-tax-badge/transactions/simple-transactions/customer-codes/). If you don't specify `avataxCustomerCode` [a dummy customer code will be used](https://developer.avalara.com/ecommerce-integration-guide/sales-tax-badge/transactions/simple-transactions/).
 
-If you want to override it, you can do so by providing a value for the user `metadata` field `avataxCustomerCode`. The app will pick it up when found on the `user` that placed the order.
+:::caution DEPRECATED
+Alternatively you can add `avataxCustomerCode` to `User` metadata. If the user is logged in, the Avatax app will use the user's id as the customer code. If the user is not logged in a dummy customer code will be used.
```

Comment:
We should add warning that it will not work for checkout_calculate_taxes hence its deprecated